resources/zib/zib-Procedure.xml

```diff
@@ -14,11 +14,12 @@
       <use value="work" />
     </telecom>
   </contact>
-  <purpose value="This Procedure resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) ]()." />
+  <description value="This profile covers the recording of therapeutic or diagnostic procedure undergone by the patient. The ServiceRequest profile (http://nictiz.nl/fhir/StructureDefinition/zib-Procedure-ServiceRequest) covers the procedure that the patient will undergo." />

```

Comment:
Missing the original concept section of the zib:

"The concept Procedure describes therapeutic or diagnostic procedure undergone by the patient or which the patient will undergo. The concept can describe in the right context historical, future as well as advised procedures.
A procedure can be a simple blood pressure measurement, but also a complex heart surgery.
In some care settings, the term treatment t is used instead of the term procedure."

Reply:
https://github.com/Nictiz/Nictiz-R4-zib2020/issues/66

Reply:
this is clear and you explain well the relationship between service request and procedure. No further comment